docs/userGuide/deployingTheSite.md

```diff
@@ -118,7 +125,13 @@ jobs:
 The sample `deploy.yml` workflow above uses the [default GitHub Token secret](https://docs.github.com/en/actions/reference/authentication-in-a-workflow) that is generated automatically for each GitHub Actions workflow. You may also use a [GitHub Personal Access Token](#generating-a-github-personal-access-token) in place of the default GitHub Token.
 </box>
 
-Once you have created the file, commit and push the file to your repo. GitHub Actions should start to build and deploy your MarkBind site. You can verify this by visiting `www.github.com/<org|username>/<repo>/actions`.
+Once you have created the file, commit and push the file to your repo. 
+
+For the first deployment on GitHub Pages, you will need to manually configure and enable GitHub Pages. 
```

Comment:
Isn't this duplicated info from line 50 above?
![image](https://github.com/MarkBind/markbind/assets/41845017/5fc24f0c-c74c-47d4-897d-e4addea4dd79)

Reply:
You're right, I also realised that it was previously mentioned aboved to manually enable github pages under `markbind deploy`. From my MarkBind set up experience, manually enabling GitHub Pages was only necessary for setting up GitHub Actions, but not required when using the command `markbind deploy`. 

Would it be better to instead add a reminder e.g. `If deploying on GitHub Pages for the first time, enable GitHub Pages as per under markbind deploy`, or completely do away with this mention?

I am unsure if this addition would be entirely relevant, as in my setup process, I jumped directly to setting up GitHub Actions instead of first using the `markbind deploy` command, which may not be the default expected workflow. I agree that if they ran `markbind deploy`, or took note of the above steps, adding this info would be unneccesary.

Reply:
> From my MarkBind set up experience, manually enabling GitHub Pages was only necessary for setting up GitHub Actions, but not required when using the command markbind deploy.

Yup you are right. I probably had the wrong impression that it needs to be done even for `markbind deploy` command.

We can move this part from `markbind deploy`:
![image](https://github.com/MarkBind/markbind/assets/41845017/7ddb8a55-9ae2-421b-bc04-e9f4b9257822)

to somewhere here under "Deploying via GitHub Actions"
![image](https://github.com/MarkBind/markbind/assets/41845017/4ad8a672-84e3-49db-bd86-1e58c0b37963)

Reply:
Also, to clarify, when you mention GitHub Actions, do you mean via the following:
```
name: Deploy MarkBind Site
on:
  push:
    branches: master

jobs:
  build:
    runs-on: ubuntu-latest
    name: test
    env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
        with:
          node-version: '16'
      - run: npm i -g markbind-cli
      - run: markbind deploy --ci
 ```
or using markbind-action ?

Reply:
Yeap! sounds good, thank you!

I was referring to the former:

> ```
> name: Deploy MarkBind Site
> on:
>   push:
>     branches: master
> 
> jobs:
>   build:
>     runs-on: ubuntu-latest
>     name: test
>     env:
>       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
>     steps:
>       - uses: actions/checkout@v3
>       - uses: actions/setup-node@v3
>         with:
>           node-version: '16'
>       - run: npm i -g markbind-cli
>       - run: markbind deploy --ci
> ```